src/bin/pg_autoctl/cli_get_set_properties.c

```diff
@@ -61,9 +68,12 @@ static CommandLine get_node_command =
 static CommandLine get_formation_number_sync_standbys =
 	make_command("number-sync-standbys",
 				 "get number_sync_standbys for a formation from the monitor",
-				 CLI_PGDATA_USAGE,
-				 CLI_PGDATA_OPTION,
-				 cli_getopt_pgdata,
+				 " [ --pgdata ] [ --json ] [ --formation ] [ --name ]",
```

Comment:
Does this one really need `--name`? Seems like it shouldn't, since `number-sync-standbys` is at the formation level.

Reply:
Right, it appears I have copy/pasted things around without enough thinking on the help strings. I went through a review of all of them now, thanks. (The fact that all just works with the default formation is a good thing, and also hides those problems when testing).